frontend/src/app/page.js

```diff
@@ -1,6 +1,22 @@
+import Link from 'next/link';
+import Image from 'next/image';
+import classes from './page.module.css';
+
 export default function Home() {
   return (
-    
-   <div>Access Nri data explorer</div>
+  <>
+    <div className={classes.topBanner}>
+      <div className={classes.logo}>
+        <Image src="/ACCESS-NRI_logo.png" alt="ACCESS-NRI_logo" fill />
+      </div>
+      <div className={`${classes.title} ${classes.dongle}`}>
+        ACCESS-NRI Data Explorer
+      </div>
+    </div>
+    <div className={`${classes.cardContainer} ${classes.dongle}`}>
+      <Link href="/catelog" className={classes.card}>Experiments Catalog</Link>
+      <Link href="/release-provenance" className={classes.card}>ACCESS-NRI Releases Provenance</Link>
```

Comment:
```suggestion
      <Link href="/release-provenance" className={classes.card}>Model Build Database</Link>
```
Is the ACCESS-NRI implied? I think I'd slightly prefer the suggested name as it is a bit more specific. The Build DB is a crucial part of the Release Provenance, but only a part.

Reply:
Sure did not know exactly what names I should give, so I just tried to infer from the content :)